### PR TITLE
Added deployment config for custom storageclass names

### PR DESCRIPTION
**Pull request by paraggit**
Added deployment config for custom storageclass names

### CODE REVIEW

File: conf/deployment/vsphere/upi_1az_rhcos_vsan_3m_3w_custom_storageclass_names.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+# This config is suppose to work on most of DCs we have.
```
Comment (petr-balogh) [inline conf/deployment/vsphere/upi_1az_rhcos_vsan_3m_3w_custom_storageclass_names.yaml:2]: I would mention here the use case of the custom storage classes.

File: conf/deployment/vsphere/upi_1az_rhcos_vsan_3m_3w_custom_storageclass_names.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  storageclassnames:
+    cephFilesystems: custom-storageclass-cephfs
+    cephObjectStores: custom-storageclass-ceph-rgw
+    cephBlockPools: custom-storageclass-blockpools
+    cephNonResilientPools: custom-ceph-nonresilientpools
+    nfs: custom-storageclass-nfs
+    encryption: custom-storageclass-encryption
+REPORTING:
+  polarion:
+    deployment_id: 'OCS-5131'
```
Comment (petr-balogh) [inline conf/deployment/vsphere/upi_1az_rhcos_vsan_3m_3w_custom_storageclass_names.yaml:25]: Missing empty line at the end of file

File: conf/deployment/vsphere/upi_1az_rhcos_vsan_3m_3w_custom_storageclass_names.yaml (hunk 1 above)
Comment (paraggit) [reply conf/deployment/vsphere/upi_1az_rhcos_vsan_3m_3w_custom_storageclass_names.yaml]: Done

File: conf/deployment/vsphere/upi_1az_rhcos_vsan_3m_3w_custom_storageclass_names.yaml (hunk 2 above)
Comment (paraggit) [reply conf/deployment/vsphere/upi_1az_rhcos_vsan_3m_3w_custom_storageclass_names.yaml]: Done

**Review by petr-balogh — DISMISSED** [review]

**Review by petr-balogh — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Pull request merged**